dotCMS/src/main/java/org/apache/velocity/tools/view/servlet/ServletToolboxManager.java

```diff
@@ -150,45 +151,37 @@ public void registerService () {
      * This method will ensure there is exactly one ServletToolboxManager
      * per xml toolbox configuration file.
      */
-    public static synchronized ServletToolboxManager getInstance(ServletContext servletContext,
-                                                                 String toolboxFile)
+    public static synchronized ServletToolboxManager getInstance(final ServletContext servletContext,
+                                                                 final String toolboxFile)
     {
-        // little fix up
-        if (!toolboxFile.startsWith("/"))
-        {
-            toolboxFile = "/" + toolboxFile;
-        }
 
         // get config file pathname
         String pathname = servletContext.getRealPath(toolboxFile);
 
         // check if a previous instance exists
-        ServletToolboxManager toolboxManager = 
-            (ServletToolboxManager)managersMap.get(pathname);
+        ServletToolboxManager toolboxManager = (ServletToolboxManager) managersMap.get(pathname);
 
         if (toolboxManager == null)
         {
             // if not, build one
-            InputStream is = null;
+            LOG.info("Trying config file '" + toolboxFile +"'");
+            InputStream is = Try.of(()->servletContext.getResourceAsStream(toolboxFile)).getOrNull();

```

Comment:
Codacy found an issue: [Avoid variables with short names like is](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5434315)